src/App.js

```diff
@@ -0,0 +1,88 @@
+import './App.css';
+import Content from './Content.js';
+import Header from './Header.js'
+import Footer from './Footer.js'
+import React, { useEffect, useState } from 'react';
+import axios from 'axios';
+import { Layout } from 'antd';
+
+const App = () => {
+
+    const [users, setUsers] = useState([]);
+    const [formVisible, setFormVisible] = useState(false);
+    const [filterString, setFilterString] = useState("");
+
+    const deleteUser = (index, id) => {
+        axios.delete(`https://jsonplaceholder.typicode.com/users/${id}`)
+            .then(res => {
+                let copyArray = users.concat();
+                copyArray.splice(index, 1);
+                setUsers(copyArray);
+            })
+    };
+
+    const handleFilter = event => {
+        setFilterString(event.target.value);
+    };
+
+    const handleAddUser = values => {
```

Comment:
This function should be part of the NewUser component. Also the passed values can be enhanced to avoid rewriting the object inside the function but this will be later discussed in another comment.